README.md

```diff
@@ -126,10 +126,10 @@ or ZMS UI corresponding to the Identityd k8s service
 ### Usage
 1. The application deployments that require Athenz identity certs need the SIA container
 and the kubernetes bound service account JWT volume mount added to the pod template
-using the sample [patch](k8s/patch/sia.yaml)
+using the sample [patch](k8s/patch/sia.json)
 
     ```
-    kubectl patch deploy <app.yaml> -p k8s-athenz-identity/k8s/patch/sia.yaml
+    kubectl patch deploy <app.yaml> -p "$(k8s/patch/sia.json)"
```

Comment:
This command is also not working, I believe you need to remove the word deploy and add `-f` and also the `"$(k8s/patch/sia.json)"` is not a command?

Reply:
This should work:
```kubectl patch -f app.yaml -p "$(cat k8s/patch/sia.json)"```

Reply:
This command actually also works with the yaml format as well.

Reply:
confirmed it is working with yaml as well.